cmd/config.go

```diff
@@ -50,16 +37,21 @@ var configCmd = &cobra.Command{
 		fmt.Println("Please specify a file prefix (nothing sensitive).")
 		prefix := prompt.Input("> ", completer)
 
-		fmt.Println("Please specify a public key to use (file path or url).")
+		fmt.Println("Please specify a public key to use (file path or url). Leave blank if intending to use SSM.")
 		pubkey := prompt.Input("> ", completer)
 
+		fmt.Println("Please specify an ssm parameter name corresponding to the public key.")
+        ssmpubkey := prompt.Input("> ", completer)
+
 		bc := options.Options{
 			Directory: dir,
 			Bucket:    bucket,
 			Org:       org,
 			Region:    region,
 			Prefix:    prefix,
 			PubKey:    pubkey,
+			SSMPubKey: ssmpubkey,
+			SSMPrivKey: ssmprivkey,
```

Comment:
Is there an option to build a config that will be used by Tempus when decrypting? If the config option is going to be used only by the partners when encrypting, do we still need SSMPrivKey in this options dictionary?

Reply:
We can think of other ways, but all this is is an object that the command line arguments get read into.  Pubkey here is only the filename of the key, where SSMpubkey and SSMprivkey are the parameter names, not the actual key values.